docs/admin/federation/index.md

```diff
@@ -0,0 +1,111 @@
+---
+---
+This guide explains how to set up a federation cluster. Federation is an alpha feature which lets you control multiple kubernetes clusters.
+[Federation proposal](https://github.com/kubernetes/kubernetes/blob/master/docs/proposals/federation.md) has more details.
+
+
+* TOC
+{:toc}
+
+## Prerequisites
+
+This guide assumes that you have a running kubernetes cluster.
+If not, then you can head over to the [getting started guides](/docs/getting-started-guides/) to bring up a cluster.
+
+This guide also assumes that you have the kubernetes source code.
```

Comment:
This guide also assumes that you have the Kubernetes source code, which you can [download here](/docs/getting-started-guides/binary_release/).